helloworld-mutual-ssl-secured/configure-certs.cli

```diff
@@ -0,0 +1,19 @@
+#configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
+/subsystem=elytron/key-store=clientKS:add(path=client.keystore.P12, relative-to=jboss.server.config.dir, credential-reference={clear-text=secret}, type=PKCS12)
+
+#generate a new key pair which will be used later to extract the certificate. This is an RSA key of size 2048. CN must be quickstartUser for the key
```

Comment:
Maybe we could reword this to just:
# Generate a new key pair for the client. We'll use an RSA key of size 2048 and we'll use CN=quickstartUser

Reply:
fixed!